File: java/문자열 계산하기.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String[] strArr = my_string.split(" ");
        answer += Integer.parseInt(strArr[0]);
        for(int i = 1; i<strArr.length; i++){
            if(i % 2 != 0){
                if(strArr[i].equals("+")){
                    answer += Integer.parseInt(strArr[i+1]);
                } else {
                    answer -= Integer.parseInt(strArr[i+1]);
                }
            }
        }
        return answer;
    }
}

File: javascript/문자열 계산하기.js
const solution = (my_string) => {
    let answer = 0;
    let strArr = [];
    strArr = my_string.split(" ");
    answer += parseInt(strArr[0]);
    for(let i = 1 ; i < strArr.length ; i++){
        if(i%2 != 0){
            if(strArr[i] == "+"){
                answer += parseInt(strArr[i+1]);
            } else {
                answer -= parseInt(strArr[i+1]);
            }
        }
    }
    return answer;
}